R/sgdnet.R

```diff
@@ -335,6 +339,19 @@ sgdnet.default <- function(x,
       grouped <- TRUE
 
       n_classes <- n_targets
+    },
+    poisson = {
+      n_classes <- 1L
+
+      if (length(unique(y)) == 1)
+        stop("only one class in response.")
```

Comment:
"Class" isn't the right name here - maybe "response is constant." would be a better phrasing.